addons/sale/wizard/res_config_settings.py

```diff
@@ -72,6 +72,8 @@ class ResConfigSettings(models.TransientModel):
 
     module_product_email_template = fields.Boolean("Specific Email")
     module_sale_amazon = fields.Boolean("Amazon Sync")
+    module_sale_shopee = fields.Boolean("Shopee Sync")
+    module_sale_lazada = fields.Boolean("Lazada Sync")
```

Comment:
I don't see any reference to Lazada in the enterprise PR. Is there a separate PR to handle it? If so I would split this PR (and update the PR/commit title and description).

Reply:
I've updated this PR and removed Lazada related code.